docs/hstaxe/installing.rst

```diff
@@ -3,23 +3,111 @@
+
+You may want to consider installing ``hstaxe`` in a new virtual or conda
+environment to avoid version conflicts with other packages you may have
+installed. We provide a premade conda environment yaml, or you may build the
+environment manually.
+
+Using the Provided HSTaXe Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: bash
+
+    conda create --name hstaxe-env --file conda_environment.yml
+    conda activate hstaxe-env
+
+
+Manually Creating Your Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Start by creating an empty conda environment:
+
+.. code-block:: bash
+
+    conda create -name hstaxe-env
+    conda activate hstaxe-env
+
+Build Prerequisites
+"""""""""""""""""""
+Because the core modules of ``hstaxe`` are written in C, we require some
+additional prerequisites to be installed before attempting to install ``hstaxe``:
+
+.. code-block:: bash
+    
+    conda install gsl cfitsio make automake autoconf libtool pkg-config -y
+    conda install wcstools -c https://conda.anaconda.org/conda-forge/ --override-channels -y
+
+
+Installing HSTaXe
+-----------------
+``hstaxe`` is distributed through PyPI. To install the latest release of ``hstaxe``:
+
+.. code-block:: bash
+
+    pip install hstaxe
+
+To instead install the latest development version, you can either install from our
+GitHub repository directly:
+
+.. code-block:: bash
+
+    pip install git+https://github.com/spacetelescope/hstaxe.git
+
+or alternatively, clone the repository locally and install the clone:
+
+.. code-block:: bash
+
+    git clone https://github.com/spacetelescope/hstaxe.git
+    cd hstaxe
+    pip install .
+
+    
+Additional Requirements
+-----------------------
+In the event the above installation instructions do not succeed, the following
+requirements, which are usually included by default, may be useful to check:
+
+C Compiler
+^^^^^^^^^^
+A C compiler suite (e.g., ``gcc`` or ``clang``) is required to build ``hstaxe``.
+These are usually included by default on most macOS and Linux distributions.
+
+C compliers are not included by default in Windows. Microsoft provides these
+tools as part of their
+`Build Tools for Visual Studio <https://visualstudio.microsoft.com/downloads>`_, 
+which can be found under "Tools for Visual Studio" towards the bottom of the page.
+
+Note that you may also need to install the
+`c-blosc <https://github.com/Blosc/c-blosc>`_ package as a dependency
+of the `tables <https://pypi.org/project/tables/>` package on certain
+platforms (e.g., this is currently needed for macOS and Windows with
+Python 3.9 due to missing ``tables`` wheels on PyPI).
+
+
+Legacy Astroconda Installation
+------------------------------
+For historical preservation, we provide the original installation instructions
+for installing ``hstaxe`` via Astroconda:
+
+.. code-block:: bash
+
+    conda install gsl cfitsio make automake autoconf libtool pkg-config -y
+    conda install wcstools -c https://ssb.stsci.edu/astroconda -y
```

Comment:
Do we need the `override-channels` flag here, in case `conda-forge` is in the user's RC file?

Reply:
Good call! Implemented in https://github.com/spacetelescope/hstaxe/pull/50/commits/7062aa0dbc0d8ac13f40de25c1ba9893a4e46a8f

Reply:
I agree that's how I assume it should work! (though our channel resolving assumptions have been wrong before! 😅 )